medical/models/medical_patient.py

```diff
@@ -101,7 +101,7 @@ def _create_vals(self, vals):
         })
         return vals
 
-    @api.model
+    @api.model_cr_context
     def _get_default_image(self, vals):
         res = super(MedicalPatient, self)._get_default_image(vals)
```

Comment:
What I mean is that (with current implementation) this 2 lines are probably useless because `_get_default_image` always refers to medical_abstract_entity._get_default_image which always returns None:
```
res = super(MedicalPatient, self)._get_default_image(vals)
if res:
    return res
```